client/my-sites/domains/domain-management/list/site-domains.jsx

```diff
@@ -224,27 +224,27 @@ export class SiteDomains extends Component {
 	}
 
 	renderDomainTableFilterButton() {
-		const { selectedSite, domains, context } = this.props;
+		const { selectedSite, domains, context, translate } = this.props;
 
 		const selectedFilter = context?.query?.filter;
 		const nonWpcomDomains = filterOutWpcomDomains( domains );
 
 		const filterOptions = [
 			{
-				label: 'Site domains',
+				label: translate( 'Site domains' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Site Domains' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/domains-filter-button-l10n)